scss/_base_blockquotes.scss

```diff
@@ -4,7 +4,7 @@
 @mixin vf-b-blockquotes {
   blockquote {
     border-left: 2px solid $color-mid-dark;
-    margin-bottom: $spv-inner--medium;
+    margin-bottom: $spv-inner--large;
```

Comment:
Why are these in this PR?

Reply:
It was a drive-by fix following the recent renaming. The actual numbers meant that medium ended up being larger than large.